stable/database/templates/statefulset.yaml

```diff
@@ -142,6 +142,11 @@ spec:
             secretKeyRef:
               name: {{ template "database.secretName" . }}
               key: database-name
+        - name: NUODOCKER_ARCHIVE_TYPE
```

Comment:
This is a good candidate for a template test. See `test/integration/template_database_test.go`.

Reply:
Done